lib/iaas/iaas.rb

```diff
@@ -16,6 +16,8 @@ def self.select_provider(env)
           return {:type => "gcp", :provider => self.init_gcp(env)}
         when "Azure"
           return {:type => "azure", :provider => self.init_azure(env)}
+        when "None"
+          return {:type => "baremetal", :provider => self.init_baremetal(env)}
```

Comment:
Can we set return as `{:type => "none", :provider => nil}`. This aligns with `when "None"` from the iaas type.